src/lib/swapper/swappers/OsmosisSwapper/OsmosisSwapper.ts

```diff
@@ -310,7 +311,9 @@ export class OsmosisSwapper implements Swapper<ChainId> {
         }),
       )
 
-    const isFromOsmo = sellAsset.assetId === osmosisAssetId
+    const isToOsmosisNetworkAsset = buyAsset.chainId === osmosisChainId
+    const isFromOsmosisNetworkAsset = sellAsset.chainId === osmosisChainId
```

Comment:
Prefer buyAsset/sellAsset terminology to stay consistent with swapper terminology